info|2020-08-07 02:05:39.584078|[INITIALIZER][initGlobalConfig],enableCompress=true,compatibilityVersion=2.5.0,versionNumber=33882112,enableStat=false,chainId=1,useSMCrypto=false
info|2020-08-07 02:05:39.584638|[INITIALIZER][SecureInitializer]loading privateKey
info|2020-08-07 02:05:39.593689|[INITIALIZER][SecureInitializer]get pub of node,nodeID=ba27c08d9702fa870f8fabf5a0a72dff6936ce9e2d92501b56c570dfcdf5d13f36ed5fd5ed692822b80d9b55529fc2c90bdf1f4e7915a4160b0fc67438f4134b
info|2020-08-07 02:05:39.593956|[INITIALIZER][SecureInitializer]use user certificate,file=conf//node.crt
info|2020-08-07 02:05:39.594355|[INITIALIZER][SecureInitializer]use ca certificate,file=conf//ca.crt
info|2020-08-07 02:05:39.594415|[INITIALIZER][SecureInitializer]use ca,file=conf//
info|2020-08-07 02:05:39.595383|[NETWORK][Host]Connecting to node,endpoint=172.17.0.2:30300
info|2020-08-07 02:05:39.595407|[NETWORK][Host]P2P StartAccept,Host=0.0.0.0:30300
info|2020-08-07 02:05:39.596191|[NETWORK][Host]Connecting to node,endpoint=172.17.0.3:30300
info|2020-08-07 02:05:39.596299|[NETWORK][Host]Connecting to node,endpoint=172.17.0.4:30300
info|2020-08-07 02:05:39.596327|[NETWORK][ASIO]asyncResolveConnect,endpoint=172.17.0.2:30300
info|2020-08-07 02:05:39.596384|[NETWORK][Host]Connecting to node,endpoint=172.17.0.5:30300
info|2020-08-07 02:05:39.596418|[NETWORK][ASIO]asyncResolveConnect,endpoint=172.17.0.3:30300
warning|2020-08-07 02:05:39.596433|[NETWORK][Host]TCP Connection refused by node,endpoint=172.17.0.2:30300,message=Connection refused
info|2020-08-07 02:05:39.596480|[P2P][Service] heartBeat,connected count=0
warning|2020-08-07 02:05:39.596500|[P2P][Service] onConnect,errorCode=5,what=Connect failed
info|2020-08-07 02:05:39.596506|[NETWORK][ASIO]asyncResolveConnect,endpoint=172.17.0.4:30300
warning|2020-08-07 02:05:39.596554|[NETWORK][Host]TCP Connection refused by node,endpoint=172.17.0.3:30300,message=Connection refused
info|2020-08-07 02:05:39.596558|[INITIALIZER][RPCInitializer],network_security.check_cert_issuer=true
warning|2020-08-07 02:05:39.596607|[P2P][Service] onConnect,errorCode=5,what=Connect failed
info|2020-08-07 02:05:39.596615|[NETWORK][ASIO]asyncResolveConnect,endpoint=172.17.0.5:30300
info|2020-08-07 02:05:39.596696|[NETWORK][Host]P2P Recv Connect, From=172.17.0.4:51302
info|2020-08-07 02:05:39.596741|[NETWORK][Host]P2P StartAccept,Host=0.0.0.0:30300
info|2020-08-07 02:05:39.601749|[NETWORK][Host]handshakeServer succ,remote endpoint=172.17.0.4:51302
info|2020-08-07 02:05:39.602869|[NETWORK][Host]startPeerSession, Remote=172.17.0.4:51302,local endpoint=172.17.0.4:30300,nodeID=ba27c08d...
info|2020-08-07 02:05:39.603320|[NETWORK][Host]handshakeClient succ,local endpoint=172.17.0.4:51302
info|2020-08-07 02:05:39.603372|[NETWORK][Host]startPeerSession, Remote=172.17.0.4:30300,local endpoint=172.17.0.4:51302,nodeID=ba27c08d...
error|2020-08-07 02:05:39.611500|[INITIALIZER][RPCInitializer],check channel_listen_port=20200,check listenIP=172.17.0.2,EINFO=Throw location unknown (consider using BOOST_THROW_EXCEPTION)
Dynamic exception type: boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::system::system_error> >
std::exception::what: bind: Cannot assign requested address

error|2020-08-07 02:05:39.611604|[INITIALIZER][Initializer]Init failed,EINFO=/FISCO-BCOS/libinitializer/RPCInitializer.cpp(122): Throw in function void dev::initializer::RPCInitializer::initChannelRPCServer(const boost::property_tree::ptree &)
Dynamic exception type: boost::exception_detail::clone_impl<dev::ListenPortIsUsed>
std::exception::what: std::exception
[dev::tag_comment*] = Please check channel_listenIP and channel_listen_port are valid

info|2020-08-07 02:05:39.611666|[INITIALIZER]ledgerInitializer stopped
info|2020-08-07 02:05:39.612707|[INITIALIZER]ChannelRPCHttpServer deleted.
info|2020-08-07 02:05:39.612731|[INITIALIZER]RPCInitializer stopped
info|2020-08-07 02:05:39.612764|[NETWORK][Host]Host exit
info|2020-08-07 02:05:39.613075|[INITIALIZER]P2PInitializer stopped
